### DISCUSSION TITLE
Community Showcase?

### DISCUSSION THREAD

**Discussion by apop880** [Elderjs/elderjs#172] [Ideas]
I think it would be an awesome idea to have a community showcase for people to share projects (and source code if they wish) that they have used Elder.js to create, along with perhaps community tutorials as well.

I've found the documentation to be extremely well-written, and comprehensive, but it's hard to account for every possible way to use Elder.js, so seeing examples of people using it in the wild can be extremely helpful, I think.

Not sure if this would fit in best as a separate page on the docs site, or perhaps in the wiki section of this repo, or on its own community-maintained site entirely. I'd be happy to get something going but didn't want to step on any toes.

**Comment by nickreese** [comment]
@apop880 No toes to step on here! Would love to help contribute. :) 

Even an awesome list would be great. If you want to take the lead on it, just let me know how I can support. I'll ping you on discord.
